test/PreAMMBatcher-e2e.ts

```diff
@@ -184,6 +184,81 @@ describe("PreAMMBatcher-e2e", () => {
     );
   });
 
+  describe("Bad encoded order", async () => {
+    it("invalid signature", async () => {
```

Comment:
ubernit:
```suggestion
    it("rejects invalid signatures", async () => {
```

Also, this would maybe be better suited as a unit test?